rust/experimental/server/src/lib.rs

```diff
@@ -17,6 +17,30 @@
 
 #![feature(impl_trait_in_assoc_type)]
 
+use std::net::{IpAddr, Ipv4Addr, SocketAddr};
```

Comment:
I'm not sure whether we should have the rule to decide the module importing sequence rule. WDYT?

Reply:
sorry, just using RustRover to format code manually